packages/mux-player/test/player.test.js

```diff
@@ -523,10 +523,14 @@ describe('<mux-player>', () => {
     const muxVideo = player.media;
 
     assert.equal(player.maxResolution, '720p');
-    assert.equal(
-      muxVideo.src,
+    const actualSrcUrl = new URL(muxVideo.src);
+    const expectedSrcUrl = new URL(
       'https://stream.mux.com/r4rOE02cc95tbe3I00302nlrHfT023Q3IedFJW029w018KxZA.m3u8?redundant_streams=true&max_resolution=720p'
     );
+    assert.equal(actualSrcUrl.searchParams.size, expectedSrcUrl.searchParams.size);
```

Comment:
NOTE: order of query params changed by refactor, so updated test to reflect what it's actually asserting: that the query param keys and values match.